chapter02/qli/src/main/java/spring/training/model/User.java

```diff
@@ -0,0 +1,23 @@
+package spring.training.model;
+
+public class User {
```

Comment:
I feel, this class should be mutable. 

- What is the point of changing values after the object creation? 
- we can use constructor injunction to populate the values.